assets/css/main.css

```diff
@@ -2613,6 +2613,10 @@ input, select, textarea {
 				width: 100%;
 			}
 
+		.image.profile-pic{
+			border-radius: 50%;
+			width: 150px;
+		}
```

Comment:
Hmm, this web project was [built with SASS](https://github.com/devedmonton/codevid19.com/tree/master/assets/sass), either we delete the SASS files or port your changes there.  Thoughts?

My bias is to use SASS but I could also see it being overkill, since we're only making small additions to the stylesheet now.

Reply:
Yes we can use SASS and port my changes there. 

If I'm understanding this correctly, did you want me to get rid of the changes in ```main.css``` and leave the changes I've made in ```_image.scss``` (the one below)?

Reply:
You got it, I didn't realize the other change was in a SASS file already 😅 .  I'm using `dart-sass` on my MacBook and when it ran, it changed the CSS structure quite a bit, so just a heads up that your diffs might be a little wonky afterward!

Reply:
No worries! I'll get rid of those changes right now.

Reply:
Oh, this is what happens when I multi-task.  I just understood what happened here.  I'm making things confusing.

This is what SASS generated, my request must've been confusing.  Let SASS generate these CSS files and commit them as needed, otherwise the website itself won't have the stylesheets.